content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md

```diff
@@ -0,0 +1,97 @@
+
+Palette's OS hardening utility performs the following tasks:
+
+<br />
+
+- Applies the latest available security updates.
+
+
+- Hardens SSH server parameters, network parameters (sysctl), and system files by ensuring proper file permissions are set.
+
+
+- Removes legacy services and graphical user interface (GUI) packages.
+
+Palette allows you to set up OS patching policies. You can patch the base OS when you deploy the cluster. Refer to the [OS Patching](/clusters/cluster-management/os-patching) documentation to learn more.
+
+<br />
+
+## Hardened Containers
+
+Container images for various application services are built using distroless images, which have significantly fewer packages and improve security by reducing attack surface.
+
+All container images are scanned for vulnerability using Palo Alto Networks Prisma Cloud (Twistlock) Defender before being published to a repository or deployed to the SaaS platform.
+
+<br />
+
+## Hardened Kubernetes
+
+Spectro Cloud has a fully automated Kubernetes verification system that adds the newest patch version of Kubernetes to its public repository. All Kubernetes packs are hardened according to CIS Benchmarks.
+
+We assess major Kubernetes versions based on the extent of changes.
+
+Kubernetes run-time security support is achieved through a variety of add-on packages, such as Sysdig Falco and Twistlock.
+
+Kubernetes cluster authentication can be optionally integrated with Kubelogin using OpenID Connect (OIDC) authentication and authorization against an external Identity Provider (IDP).
+
+You can set a schedule to start Kubernetes conformance and compliance tests using kube-bench, kube-hunter, and Sonobuoy. These tests ensure tenant clusters are secure, compliant, and up to date.
+
+<br />
+
+## Kubernetes Authentication & Authorization
+
+Kubernetes cluster authentication can be optionally integrated with Kubelogin with OIDC-based authentication/authorization against external IDP. This enables group membership-based access control on different namespaces within the tenant Kubernetes cluster. Spectro Cloud’s Terraform provider also supports automatically set role binding on namespaces with users or groups.
+
+<br />
+
+## Compliance & Security Scans
+
+Palette provides a way to run compliance, security, conformance, and software bill of materials (SBOM) scans on tenant clusters. These scans ensure clusters adhere to specific compliance and security standards. The scans also detect potential vulnerabilities by performing penetration tests.
+
+Palette supports four types of scans. Each scan generates reports with details specific to the type of scan. You can initiate multiple scans of each type over time. In addition, Palette keeps a history of previous scans for comparison purposes.
```

Comment:
What are the four types of scans? Are they compliance, security, conformance, and SBOM? 
If so, line 97 repeats. You could delete the first sentence and add "You can initiate multiple scans..." to line 95.